README.md

```diff
@@ -226,38 +226,6 @@ NOT TODO
 -
 - provide a _format_ flag - just use the awesome [jq](https://stedolan.github.io/jq/)
 
-CONTRIBUTING
```

Comment:
Should we add a link here to the contributing page?

Reply:
Hmm, I don't see the need for it. I think that the file CONTRIBUTING.md in the top directory is prominent enough.

BUT, at least c/image, c/storage and c/libpod have such a link. So I'll add it for consistency reasons.

Reply:
Done